src/System.ComponentModel.EventBasedAsync/tests/BackgroundWorkerTests.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+public class BackgroundWorkerTests
+{
+    private static ManualResetEventSlim manualResetEvent1 = new ManualResetEventSlim(false);
```

Comment:
Any reason we use static state like this rather than just having these be local to the test and using lambdas to capture them as necessary?  Static state makes it much more likely that we'll run into random test failures as we accidentally leak details across tests, forgot to reset static state, etc.